README.md

```diff
@@ -76,23 +76,16 @@ to 1_apples
 ```js
 exports.migrate = function(client, done) {
     var db = client.db;
-    db.card.insert( { id: 1, type: 'apple', color: 'red' }, (err, results) => {
 
-        if (err) return done(err);
-
-        return done(null);
-    });
+    db.card.insert( { id: 'red-apple', type: 'apple', color: 'red' })
+        .then(() => done(null), done);
```

Comment:
Should we get a `nodebackify` function to handle these promise chains? I don't mind dragging one in from npm, but although I'm a little uncertain, I think we need to make sure that if a promise is rejected without a reason (i.e. with an `undefined` or a `null`), that we `callback` with _some_ error, so that the consumer actually goes down the `if (err)` code path.

Note, that we should also make sure to `callback()` inside a `setTimeout()`, because if the _callback_ throws - the promise will swallow the exception and we'll get an unhandled rejection.

Reply:
Apparently https://nodejs.org/api/util.html#util_util_callbackify_original is available in node 8.2+ and does all of the above.

Reply:
I like the sound of `util.callbackify` but that would mean only supporting node 8.2+
Are we happy with this constraint considering east supports node +0.10?

Reply:
I think supporting node 0.x is nuts at this day and age - even node 4 will stop being support in a month or two, so really only node 6 and 8 are worth the support, but I think all the users of this library are on node 8 by now ;) And if they are not - then they should probably contribute to the development of this library.

Reply:
I commented on the README and I thought I was commenting on the code 🤦‍♂️ 

I suppose we could update the README to do async/await examples? Eventually.